front/app/containers/Admin/projects/project/files/components/FilePreview/CsvFilePreview/index.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { useEffect, useState, useRef } from 'react';
+
+import {
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  Spinner,
+  Text,
+  Box,
+} from '@citizenlab/cl2-component-library';
+import Papa from 'papaparse';
```

Comment:
Note: I added two new libraries in this PR. Both are very small and appear well used and have been updated recently:

Papaparse: https://www.npmjs.com/package/papaparse
react-markdown: https://www.npmjs.com/package/react-markdown